tests/foreman/api/test_user.py

```diff
@@ -631,10 +630,10 @@ def test_positive_ssh_key_in_host_enc(self):
 
 
 @pytest.mark.run_in_one_thread
+@pytest.mark.skip_if_not_set('ldap')
 class TestActiveDirectoryUser:
     """Implements the LDAP auth User Tests with Active Directory"""
 
-    @skip_if_not_set('ldap')
```

Comment:
Context might not be correct in my view, but I don't see a mark replacement for this and others in this file

Reply:
Its marked on the class now.